apps/src/code-studio/initializeCodeMirror.js

```diff
@@ -41,6 +44,26 @@ module.exports = function (target, mode, callback, attachments) {
   if (mode === 'markdown') {
     backdrop = mode;
     mode = 'spell-checker';
+
+    // In markdown mode, look for a preview element (found by just appending
+    // _preview to the target id), if it exists extend our callback to update
+    // the preview element with the markdown contents
+    const previewElement = document.getElementById(target + '_preview');
+    if (previewElement) {
+      const originalCallback = callback;
+      updatePreview = editor => {
+        $(previewElement).html(marked(editor.getValue()));
+        $(previewElement).children('details').details();
```

Comment:
I know you've already got the element, but isn't it a code smell to jQuery-wrap the same element multiple times?  Especially in a callback here.  Could just use jQuery to locate the element on line 51 and reuse the jQuery object in the callback.